sdk/dapp-kit/src/components/WalletProvider.tsx

```diff
@@ -52,6 +53,7 @@ export function WalletProvider({
 	storageAdapter = localStorageAdapter,
 	storageKey = DEFAULT_STORAGE_KEY,
 	enableUnsafeBurner = false,
+	autoConnect = false,
```

Comment:
Is this a default switch or am I going crazy

Reply:
Hahaha you are not crazy 😆. I thought it might be a bit better to have folks explicitly opt-in to the auto-connect functionality when I originally set up the component, but eh maybe we want to revert this back?

Reply:
I’m open to this, just wanted to make sure

Reply:
Yep, that's a good idea. Glad to help out with that or take that on myself once the rest of this work is completed 😄